ts/event.ts

```diff
@@ -5,28 +5,40 @@ import { modalState } from "./state.js";
 const TIME_VALIDATION_ERROR_MESSAGE = "Event cannot end before it starts.";
 const TITLE_VALIDATION_ERROR_MESSAGE = "Please enter a title.";
 
-const eventModal = document.querySelector("#eventModal");
-const eventForm = document.querySelector("#eventForm");
-const eventButton_create = document.querySelector(".create");
-const eventButton_cancel = document.querySelector("#event-cancel");
-const eventButton_save = document.querySelector("#event-save");
-const eventDate = document.querySelector("#event-date");
-const startTime = document.querySelector("#event-startTime");
-const endTime = document.querySelector("#event-endTime");
-const title = document.querySelector("#event-title");
-
-const convertInputToDate = (dateString, timeString) => {
+const eventModal = document.querySelector("#eventModal") as HTMLDivElement;
+const eventForm = document.querySelector("#eventForm") as HTMLFormElement;
+const eventButton_create = document.querySelector(
```

Comment:
Stick with a naming convention.
Should be `eventButtonCreate`, `eventButtonCancel`....
Also, judging by the context it could also be `createEventButton`, `cancelEventButton`...